examples/flutter_gallery/lib/demo/text_field_demo.dart

```diff
@@ -73,55 +75,55 @@ class TextFieldDemoState extends State<TextFieldDemo> {
         title: new Text('Text fields')
       ),
       body: new Form(
+        key: _formKey,
         child: new Block(
           padding: const EdgeInsets.symmetric(horizontal: 16.0),
           children: <Widget>[
-            new Input(
-              hintText: 'What do people call you?',
-              labelText: 'Name',
-              formField: new FormField<String>(
-                // TODO(mpcomplete): replace with person#name=
-                setter:  (String val) { person.name = val; },
-                validator: _validateName
-              )
+            new FormField<InputValue>(
```

Comment:
I added a comment - I want to demonstrate the equivalence of the 2 ways of doing it.